projects/aws/eks-charts/aws-for-fluent-bit/Makefile

```diff
@@ -0,0 +1,41 @@
+AWS_FOR_FLUENT_BIT_CHART_REPO?=eks-charts
+AWS_FOR_FLUENT_BIT_CHART_REPO_URL?=https://github.com/aws/$(AWS_FOR_FLUENT_BIT_CHART_REPO).git
+AWS_FOR_FLUENT_BIT_CHART_GIT_TAG?=v0.0.41
+
+ifeq ("$(AWS_FOR_FLUENT_BIT_CHART_REPO_URL)","")
+        $(error No repository URL was provided.)
```

Comment:
Is this indented different than the others?

Reply:
Fixed